steward_abi_build/src/main.rs

```diff
@@ -9,22 +9,21 @@ fn main() {
         ("CellarV1", "cellar_v1"),
         ("CellarV2", "cellar_v2"),
         ("CellarV2_2", "cellar_v2_2"),
```

Comment:
Is this affected by the 2dot2 change or is that just a proto thing?

Reply:
Proto enum thing